### PR TITLE
Features/snpeff

### PR DESCRIPTION
**Pull request by northwestwitch**
Moved SnpEff to MIP::Program::Variantcalling. Tested on rasta

### CODE REVIEW

File: lib/MIP/Program/Variantcalling/SnpEff.pm [hunk 1]
```diff
@@ -1,187 +1,141 @@
...
-##          : $stdoutfile_path         => Stdoutfile path
+## Arguments: $config_file_path       => Config file path
+##          : $FILEHANDLE             => Filehandle to write to
+##          : $genome_build_version   => Genome build version
+##          : $infile_path            => Infile path
+##          : $outfile_path           => Outfile path
+##          : $stderrfile_path        => Stderrfile path
+##          : $stderrfile_path_append => Append stderr info to file path
+##          : $stdoutfile_path        => Stdoutfile path
+##          : $verbosity            => Increase output verbosity
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/SnpEff.pm]: Align fat comma

File: lib/MIP/Program/Variantcalling/SnpEff.pm [hunk 2]
```diff
@@ -1,188 +0,0 @@
...
-
-    # Set the version for version checking
-    our $VERSION = 1.0.2;
-
-    # Functions and variables which can be optionally exported
-    our @EXPORT_OK = qw{ snpeff_download };
-
-}
-
-sub snpeff_download {
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/SnpEff.pm]: This one is used in mip_install. Please add it again in your Snpeff.pm and make sure that you change the pm path correctly for mip_install

Comment (northwestwitch) [reply lib/MIP/Program/Variantcalling/SnpEff.pm]: Whoa, where does it come from? I didn't even realize I removed it! But yes, I'll add it again!

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**